zds/searchv2/views.py

```diff
@@ -190,102 +204,124 @@ def get_queryset(self):
             if self.search_form.cleaned_data["from_library"] == "on":
                 self.from_library = True
 
-            # Setting the different querysets (according to the selected models, if any)
-            part_querysets = []
-            chosen_groups = self.search_form.cleaned_data["models"]
+            # Check which collections needs to search
```

Comment:
```suggestion
            # Check in which collections search is performed
```
c'est bien ce que vous voulez dire ?

Reply:
Oui c'est bien ça, merci.